### ISSUE TITLE
Adding fill-extrusion to <Layer/>

### ISSUE THREAD

**Issue by Utzel-Butzel** [alex3165/react-mapbox-gl#310]
Currently only Geojson supports 3D extrusions (at least I think):

`type?: 'symbol' | 'line' | 'fill' | 'circle' | 'raster';
`

Many thanks for this great piece of software! :)

**Comment by alex3165** [comment]
Hey @Utzel-Butzel did you have a look at https://github.com/alex3165/react-mapbox-gl/blob/master/example/src/demos/threeDMap.tsx ?

**Comment by alex3165** [comment]
Layer should support all layer types :
```ts
"fill" | "line" | "symbol" | "circle" | "fill-extrusion" | "raster" | "background"
```

**Comment by alex3165** [comment]
Released from tag `3.0.0`
